### PR TITLE
cmd：fix info

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: cmd/oci-image-validate/oci-image-validate.1.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ oci-image-validate \- Validate one or more image files
   Only applicable if type is image or imageLayout.
 
 **--type**=""
-  Type of the file to validate. If unset, oci-image-validate will try to auto-detect the type. One of "imageLayout,image,manifest,manifestList,config"
+  Type of the file to validate. If unset, oci-image-validate will try to auto-detect the type. One of "imageLayout,image,manifest,imageIndex,imageConfig"
```
Comment (jonboulle) [inline cmd/oci-image-validate/oci-image-validate.1.md]: https://github.com/opencontainers/image-tools/blob/374a7f6c1a485da6d5d23529f3456ff4721b74ac/image/autodetect.go#L34 isn't changed

Comment (q384566678) [inline cmd/oci-image-validate/oci-image-validate.1.md]: updated, PTAL

**Comment by jonboulle** [comment]
LGTM

**Comment by xiekeyang** [comment]
LGTM

**Pull request merged**